### PR TITLE
Establish export pattern for importing individual controls

### PR DESCRIPTION
**Pull request by NathanFarmer**
- Establish export pattern for importing individual controls
- Reorganize Numeric controls
- Rename `AlertControl` to `AlertLayout`

### CODE REVIEW

File: src/renderers.ts [hunk 1]
```diff
@@ -66,19 +66,19 @@ export const rendererRegistryEntries: JsonFormsRendererRegistryEntry[] = [
     renderer: withJsonFormsLabelProps(AlertControl),
   },
   {
-    tester: rankWith(2, isNumericControl),
-    renderer: withJsonFormsControlProps(NumericControl),
-  },
-  {
-    tester: rankWith(2, isNumericControl),
-    renderer: withJsonFormsControlProps(NumericControl),
-  },
```
Comment (NathanFarmer) [inline src/renderers.ts:75]: Must be a bad merge here

File: src/renderers.ts [hunk 2]
```diff
@@ -66,19 +66,19 @@ export const rendererRegistryEntries: JsonFormsRendererRegistryEntry[] = [
...
-    tester: rankWith(3, isNumericSliderControl),
+    tester: rankWith(
+      3,
+      and(
+        or(isNumberControl, isIntegerControl),
+        schemaMatches((schema: JsonSchema) => {
+          return schema.minimum !== undefined && schema.maximum !== undefined
+        }),
+      ),
+    ),
```
Comment (NathanFarmer) [inline src/renderers.ts:81]: This is short enough that I decided to just move it in here with everything else

File: src/index.ts [hunk 3]
```diff
@@ -1,2 +1,13 @@
 export { rendererRegistryEntries, cellRegistryEntries } from "./renderers"
 export type * from "./ui-schema"
+
+export { BooleanControl } from "./controls/BooleanControl"
+export { AlertControl } from "./controls/AlertControl"
```
Comment (DrewHoo) [inline src/index.ts:5]: As part of these changes, can you rename this to AlertLayout (I think that's the best name, but it's possible this is best described as a cell) ... getting names right is crucial bc changing names afterward is a breaking change

**Review by DrewHoo — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 3 above)
Comment (NathanFarmer) [reply src/index.ts]: Makes sense to me! ✅

**Review by DrewHoo — APPROVED** [review]

**Pull request merged**